### PR TITLE
ServiceNow page fixes

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

### :nut_and_bolt: Description: What code changed, and why?

This PR fixes a broken link within the ServiceNow documentation as reported internally. I also took this opportunity to apply some brief copyediting and formatting fixes.

### :chains: Related Resources

N/A

### :+1: Definition of Done

🟢  means go! :)

### :athletic_shoe: How to Build and Test the Change

Use the Netlify link below to see the changes on the ServiceNow Integration page.

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/docs-chef-io/content/automate/servicenow_integration_install.md [hunk 1]
```diff
@@ -24,14 +22,15 @@ The integration between a Chef Automate server and a ServiceNow instance require
 * Chef Automate
 * Chef Automate Incident Creation Application in ServiceNow
 
-The Chef Automate Incident Creation application is a ServiceNow certified scoped application available from the ServiceNow store. [Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance. Once configured, the Chef Automate server sends HTTPS JSON notifications to the Chef Automate Incident Creation Application in a ServiceNow instance to create or update incidents.
+The Chef Automate Incident Creation application is a ServiceNow certified scoped application available from the ServiceNow store. [Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/servicenow_integration_install.md:25]:
```suggestion
[Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance.
The Chef Automate Incident Creation application is a ServiceNow certified scoped application available from the ServiceNow store.
```

Comment (kagarmoe) [reply components/docs-chef-io/content/automate/servicenow_integration_install.md]: Move from the general to the specific & also stay consistent with the bullet points.

File: components/docs-chef-io/content/automate/servicenow_integration_install.md [hunk 2]
```diff
@@ -24,14 +22,15 @@ The integration between a Chef Automate server and a ServiceNow instance require
...
+Once configured, the Chef Automate server sends HTTPS JSON notifications to the Chef Automate Incident Creation Application in a ServiceNow instance to create or update incidents.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/servicenow_integration_install.md:27]:
```suggestion
That sends HTTPS JSON notifications from your Chef Automate installation to the ServiceNow Chef Automate Incident Creation Application to create or update incidents.
```
Add to the preceding paragraph & edit how they come together.

**Review by kagarmoe — CHANGES_REQUESTED** [review]
1 ) Need to bump up the header levels after removing the h2
2) Minor copy change

File: components/docs-chef-io/content/automate/servicenow_integration_install.md (hunk 2 above)
Comment (mjingle) [reply components/docs-chef-io/content/automate/servicenow_integration_install.md:27]: @kagarmoe Do you want to add or replace line 27? Also what does "that" refer to in your suggested sentence?

Comment (kagarmoe) [reply components/docs-chef-io/content/automate/servicenow_integration_install.md:27]: I'm leaving that to you with `& edit how they come together`.

File: components/docs-chef-io/content/automate/servicenow_integration_install.md [hunk 3]
```diff
@@ -13,25 +13,25 @@ draft = false
...
 
 The integration between a Chef Automate server and a ServiceNow instance requires the following:
 
 * Chef Automate
 * Chef Automate Incident Creation Application in ServiceNow
 
-The Chef Automate Incident Creation application is a ServiceNow certified scoped application available from the ServiceNow store. [Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance. Once configured, the Chef Automate server sends HTTPS JSON notifications to the Chef Automate Incident Creation Application in a ServiceNow instance to create or update incidents.
+[Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance.
+The Chef Automate Incident Creation Application is a ServiceNow certified scoped application available from the ServiceNow store.
+The ServiceNow Chef Automate Incident Creation Application creates or updates incidents based on HTTPS JSON notifications sent from your Chef Automate installation.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/servicenow_integration_install.md:27]: The app gets called by three different names in this section:

- Chef Automate ServiceNow Integration
- Chef Automate Incident Creation Application
- ServiceNow Chef Automate Incident Creation Application

Settle on one way of naming it and enforce that wherever it is used in this doc.

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: components/docs-chef-io/content/automate/servicenow_integration_install.md (hunk 3 above)
Comment (mjingle) [reply components/docs-chef-io/content/automate/servicenow_integration_install.md:27]: You bring up a great point. Let me add to the confusion in wording and note that we have two apps in the ServiceNow Store named:
- [Chef Automate Integration App](https://store.servicenow.com/sn_appstore_store.do#!/store/application/02bf0043dba2d30070cd5e25ca9619a3/1.0.24?referer=%2Fstore%2Fsearch%3Flistingtype%3Dallintegrations%25253Bancillary_app%25253Bcertified_apps%25253Bcontent%25253Bindustry_solution%25253Boem%25253Butility%26q%3Dautomate&sl=sh)
- [Chef Automate Incident Creation](https://store.servicenow.com/sn_appstore_store.do#!/store/application/02bf0043dba2d30070cd5e25ca9619a3/1.0.24?referer=%2Fstore%2Fsearch%3Flistingtype%3Dallintegrations%25253Bancillary_app%25253Bcertified_apps%25253Bcontent%25253Bindustry_solution%25253Boem%25253Butility%26q%3Dautomate&sl=sh)

Comment (mjingle) [reply components/docs-chef-io/content/automate/servicenow_integration_install.md:27]: I confirmed that this documentation is specifically for the Chef Automate Incident Creation application.

**Review by kagarmoe — APPROVED** [review]
LGTM
tenor-145202506

**Pull request merged**